spharpy/samplings/coordinates.py

```diff
@@ -310,3 +310,17 @@ def n_max(self, value):
             self._n_max = None
         else:
             self._n_max = int(value)
+
+    @property
```

Comment:
Note: This is reviewed in #120 but also required here. This PR relies on the property but does not care how it is implemented.